### ISSUE TITLE
1.9.4 Support - NoClassDefFoundError IChatBaseComponent

### ISSUE THREAD

**Issue by mibby** [Siguza/VirtualPack#37]
VirtualPack 2.8.4
Spigot MC 1.9.4

@Siguza VirtualPack needs to be updated to support MC/Spigot 1.9.4. Simple as just compiling against 1.9.4-R0.1-SNAPSHOT and v1_9_R2 instead of v1_9_R1? Entity human nms in enchanttable also needs to be changed from cQ to cR?

```
[15:03:37] [Server thread/INFO]: [VirtualPack] Enabling VirtualPack v2.8.4
[15:03:37] [Server thread/ERROR]: Error occurred while enabling VirtualPack v2.8.4 (Is it up to date?)
java.lang.NoClassDefFoundError: net/minecraft/server/v1_9_R1/IChatBaseComponent
	at net.drgnome.virtualpack.util.ConfigProxy.setDefs(ConfigProxy.java:137) ~[?:?]
	at net.drgnome.virtualpack.util.ConfigProxy.<init>(ConfigProxy.java:91) ~[?:?]
	at net.drgnome.virtualpack.util.Config.reload(Config.java:27) ~[?:?]
	at net.drgnome.virtualpack.VPlugin.reloadConfig(VPlugin.java:905) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.getConfig(JavaPlugin.java:163) ~[spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at net.drgnome.virtualpack.util.Config.reload(Config.java:27) ~[?:?]
	at net.drgnome.virtualpack.VPlugin.init(VPlugin.java:87) ~[?:?]
	at net.drgnome.virtualpack.VPlugin.onEnable(VPlugin.java:264) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) ~[spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at org.bukkit.craftbukkit.v1_9_R2.CraftServer.loadPlugin(CraftServer.java:361) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at org.bukkit.craftbukkit.v1_9_R2.CraftServer.enablePlugins(CraftServer.java:321) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at net.minecraft.server.v1_9_R2.MinecraftServer.t(MinecraftServer.java:416) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at net.minecraft.server.v1_9_R2.MinecraftServer.l(MinecraftServer.java:381) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at net.minecraft.server.v1_9_R2.MinecraftServer.a(MinecraftServer.java:336) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at net.minecraft.server.v1_9_R2.DedicatedServer.init(DedicatedServer.java:268) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at net.minecraft.server.v1_9_R2.MinecraftServer.run(MinecraftServer.java:532) [spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_92]
Caused by: java.lang.ClassNotFoundException: net.minecraft.server.v1_9_R1.IChatBaseComponent
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:91) ~[spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:86) ~[spigot.jar:git-Spigot-4af49dc-c5e9a16]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_92]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_92]
	... 19 more
```

**Comment by Siguza** [comment]
Should be fixed in 6907bf1472875ab4b7ef79fcca0b85fb20507069.

**Comment by mibby** [comment]
All seems fine, thanks! :)
